aci/convert/convert_test.go

```diff
@@ -712,6 +715,151 @@ func TestConvertContainerGroupStatus(t *testing.T) {
 	assert.Equal(t, "Unknown", GetStatus(container(nil), group(nil)))
 }
 
+func TestConvertSecrets(t *testing.T) {
```

Comment:
If possible, would be nice to extract the secrets bits from convert.go to secrets.go, and same for the unit tests